Source/Router.swift

```diff
@@ -118,6 +118,7 @@ public final class Router {
         case post = "POST"
         case put = "PUT"
         case delete = "DELETE"
+		case patch = "PATCH"
```

Comment:
@joanromano I think we misuse the review... you add comments but people won't see it until you "finish" the review with one of the options (comment, approve, decline)...
so we couldn't see your comments until you approved 👎